### ISSUE TITLE
SUN RGB-D Data Preparation

### ISSUE THREAD

**Issue by Divadi** [cheng052/BRNet#2]
Hello, thank you for open-sourcing your codebase.

I was wondering whether your training data for SUN RGB-D was v1 or v2. It is mentioned in the paper that results shown are from the v1 validation set, but I wasn't sure which version you were using for training.

**Comment by cheng052** [comment]
Hi, thanks for your interest in our work.

Cause we reimplement our method using mmdetection3d codebase, we just use the default data preparation setting in mmdetection3d, which uses the SUN RGB-D v2 label(both train and validation). 
The original code for CVPR submission uses SUN RGB-D v1 label as in the original VoteNet codebase(both train and validation). 
It seems the detection results are comparable, and not affected by the label version.
Thanks for pointing out this!
